hieradata/development.yaml

```diff
@@ -101,6 +101,7 @@ govuk::node::s_development::apps:
 govuk_app_enable_capistrano_layout: false
 govuk_app_enable_services: false
 
+govuk::apps::email_campaign_frontend::vhost: 'draft-email-campaign-frontend'
```

Comment:
This is specific to draft, which might be confusing (I think non-draft would be better as a default).